packages/Sandblocks-Core/Rectangle.extension.st

```diff
@@ -10,6 +10,20 @@ Rectangle >> asEdgeInsets [
 		bottom: self bottom
 ]
 
+{ #category : #'*Sandblocks-Core' }
+Rectangle >> distanceTo: aRectangle [
+
+	^ ((aRectangle topLeft - self bottomRight) max: 0@0) - ((self topLeft - aRectangle bottomRight) max: 0@0).
+	
+]
+
+{ #category : #'*Sandblocks-Core' }
+Rectangle >> distanceToPoint: aPoint [
+
+	^ ((aPoint - self bottomRight) max: 0@0) - ((self topLeft - aPoint) max: 0@0).
+	
```

Comment:
These two are approximations right? Or am I missing a magic trick somewhere? :D 

If they are approximations, maybe change the name accordingly. For an exact (but probably slow) way you could consider using `Rectangle>>#pointNearestTo:`

Reply:
Ah it returns a Point, I would tend to change the name then, just because I feel like other packages could very feasibly implement the same selectors but with entirely different behavior